config/jobs/kubernetes/sig-node/dynamic-resource-allocation.yaml

```diff
@@ -5,9 +5,11 @@ periodics:
     cluster: eks-prow-build-cluster
     interval: 6h
     annotations:
-      testgrid-dashboards: sig-node-dynamic-resource-allocation
-      testgrid-tab-name: ci-kind-dra
-      testgrid-alert-email: patrick.ohly@intel.com
+      testgrid-dashboards: sig-release-master-blocking, sig-node-release-blocking, sig-node-dynamic-resource-allocation
```

Comment:
From sig-node side, I am happy with "sig-node-release-blocking" but I think we need input from sig-release for sig-release-master-blocking.

Maybe we can consider "sig-release-master-informing" first and let it soak for a release or two.

Reply:
> Maybe we can consider "sig-release-master-informing" first and let it soak for a release or two.

this is the usual approach (at least one release of informing).

Reply:
Agree, see also [Slack](https://kubernetes.slack.com/archives/C2C40FMNF/p1734417601687079).

I've reduced this PR to just the job cleanups, without changing the status of the jobs. Let's merge it now and in parallel start the process for release informing.

/hold cancel